web/src/pages/orders/hooks.js

```diff
@@ -0,0 +1,41 @@
+import { useCategories } from "@/hooks/redux/useCategories";
+import { useOrders } from "@/hooks/redux/useOrders";
+
+const useHooks = () => {
+  const { orders, isLoading: ordersLoading } = useOrders();
+  const { categories, isLoading: categoriesLoading } = useCategories();
+
+  const isLoading = ordersLoading || categoriesLoading;
+
+  const getCategoryLabel = (order) => {
+    const category = categories.find(cat => cat.id === order.category_id);
+    return category ? category.label : "Unknown Category";
+  };
+
+  const capitalizeFirstLetter = (string) => {
+    return string.charAt(0).toUpperCase() + string.slice(1);
+  };
+
+  const formattedOrders = orders.map((order) => [
+    order.brand,
+    new Date(order.created_at).toLocaleString("en-US", {
+      month: "long",
+      day: "numeric",
+      year: "numeric",
+    }),
```

Comment:
transfer to web/src/hooks/lib/util.js
so we can use it on other page in the future

```suggestion
 // inside web/src/hooks/lib/util.js
  export const capitalizeFirstLetter = (string) => {
    return string.charAt(0).toUpperCase() + string.slice(1);
  };
  
  export formatDate = (date) => {
  return new Date(date).toLocaleString("en-US", {
      month: "long",
      day: "numeric",
      year: "numeric",
    })
  }


 // in your hooks only
  const formattedOrders = orders.map((order) => [
    order.brand,
    formatDate(order.created_at),
```